<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1353</width>
    <height>682</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="cbbLoc2">
   <property name="geometry">
    <rect>
     <x>1188</x>
     <y>20</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>20</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Hi·ªÉn th·ªã danh s√°ch theo:</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>1291</width>
     <height>451</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnXoa">
   <property name="geometry">
    <rect>
     <x>1230</x>
     <y>610</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>üóë X√≥a</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSua">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>610</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>‚úèÔ∏è S·ª≠a</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnThem">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>610</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>‚ûï Th√™m</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Qu·∫£n l√Ω ƒëo√†n vi√™n</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>530</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTimKiem">
   <property name="geometry">
    <rect>
     <x>1230</x>
     <y>530</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>üîç T√¨m ki·∫øm</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>530</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nh·∫≠p m√£ ho·∫∑c t√™n ƒëo√†n vi√™n:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbbLoc1">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>20</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
